packages/shared/src/components/search/SearchBar.tsx

```diff
@@ -1,264 +1,46 @@
-import TimerIcon from '../icons/Timer';
-import { IconSize } from '../Icon';
-import { SearchProgressBar } from './SearchProgressBar';
 import {
   SearchBarSuggestion,
   SearchBarSuggestionProps,
 } from './SearchBarSuggestion';
 import AuthContext from '../../contexts/AuthContext';
-import { SimpleTooltip } from '../tooltips/SimpleTooltip';
 import useSidebarRendered from '../../hooks/useSidebarRendered';
+import { SearchBarInput, SearchBarInputProps } from './SearchBarInput';
 
-export interface SearchBarProps
-  extends Pick<
-    InputHTMLAttributes<HTMLInputElement>,
-    | 'placeholder'
-    | 'value'
-    | 'style'
-    | 'name'
-    | 'autoFocus'
-    | 'onBlur'
-    | 'onFocus'
-    | 'aria-haspopup'
-    | 'aria-expanded'
-    | 'onKeyDown'
-    | 'disabled'
-    | 'readOnly'
-    | 'aria-describedby'
-    | 'autoComplete'
-  > {
-  inputId: string;
-  valueChanged?: (value: string) => void;
-  showIcon?: boolean;
-  rightButtonProps?: ButtonProps<'button'> | false;
-  completedTime?: string;
-  showProgress?: boolean;
-  className?: {
-    container?: string;
-  };
-  onSubmit?: <T>(event: MouseEvent<T>) => void;
-}
-
-export interface SearchBarProgressBarProps {
-  progress: number;
-}
+export type SearchBarProps = Pick<SearchBarInputProps, 'className'>;
 
-export const SearchBar = forwardRef(function SearchBar(
-  {
-    inputId,
-    name,
-    value,
-    valueChanged,
-    placeholder = 'Ask anything…',
-    readOnly,
-    className,
-    autoFocus,
-    disabled,
-    rightButtonProps = { type: 'button' },
-    'aria-describedby': describedBy,
-    onBlur: externalOnBlur,
-    onFocus: externalOnFocus,
-    showProgress = true,
-    completedTime,
-    onSubmit: handleSubmit,
-    ...props
-  }: SearchBarProps,
-  ref: ForwardedRef<HTMLDivElement>,
-): ReactElement {
-  const {
-    inputRef,
-    focused,
-    hasInput,
-    onFocus,
-    onBlur,
-    onInput,
-    focusInput,
-    setInput,
-  } = useInputField(value, valueChanged);
+export function SearchBar({ className }: SearchBarProps): ReactElement {
   const { user, showLogin } = useContext(AuthContext);
-  const progress = 0;
-  const searchHistory = [];
   const { sidebarRendered } = useSidebarRendered();
   const suggestions: SearchBarSuggestionProps[] = [];
 
   if (!user) {
-    suggestions.push({
-      suggestion:
-        'Sign up and read your first post to get search recommendations',
-      onClick: () => showLogin('search bar suggestion'),
-    });
+    suggestions.push(
+      {
+        suggestion:
+          'Sign up and read your first post to get search recommendations',
+        onClick: () => showLogin('search bar suggestion'),
+      },
+      {
+        suggestion: 'Sign up and read your ',
+        onClick: () => showLogin('search bar suggestion'),
+      },
+      {
+        suggestion:
+          'Sign up and read your first post to get search recommendations',
+        onClick: () => showLogin('search bar suggestion'),
+      },
+    );
   }
```

Comment:
This should only have one right?

Reply:
Oh, right. Was testing some stuff. Let me fix it up.